hook/hook.go

```diff
@@ -10,15 +10,23 @@ import (
 	"github.com/hashicorp/go-multierror"
 )
 
+// Response defines the handler's return value
+// - Sets http.StatusOK and payload if you want the coresponding sink list to process the payload.
+// - Sets other 2xx code if you want to short-circuit the processing, but you don't want to indicate
+//   an error (e.g. skip the processing). You can optionally set a detail to explain the reason.
+//   This will show up on the webhook sender's page.
+// - Sets other HTTP code and err if you do want to indicate an error.
+type Response struct {
+	httpCode int
```

Comment:
nit: Why not call it `statusCode`, seems more obvious?

```suggestion
	statusCode int
```

Reply:
status and code are synonyms, I intend to make it explicit that it's http code